example/passthrough_hp.cc

```diff
@@ -1245,6 +1298,14 @@ static cxxopts::ParseResult parse_options(int argc, char **argv) {
     fs.nosplice = options.count("nosplice") != 0;
     fs.num_threads = options["num-threads"].as<int>();
     fs.clone_fd = options["clone-fd"].as<bool>();
+
+    fs.bypass_rw = options["bypass-rw"].as<int>();
+    if (fs.bypass_rw) {
+        std::cout << "===== Warning RW IO bypassed! Data corruption expected! ====="
+                  << std::endl;
+        fs.bypass_buf = reinterpret_cast<char *>(new uint8_t [1024 * 1024 * 2]);
```

Comment:
Can we document why this is the right buffer size, and check for out of bounds writes and reads in the callbacks?